components/CommonCompHabitChart.js

```diff
@@ -85,6 +87,26 @@ const CommonCompHabitChart = ({ groupMembersData, groupMembersNames, groupColor
     }
     //console.log(userDataList)
 
+    //calculatestreak
+    var streakLength = 0;
+    if (userDataList.length > 0){
```

Comment:
Hmmmm I am wondering if what will happen if userDataList[i] has all 0s. Should we reset streak to 0?

Reply:
This just checks that the list of arrays isn't empty. The streak should still reset to 0 if none logged any data